### PR TITLE
Add data input form and sidebar

### PR DESCRIPTION
**Pull request by tinashechiraya**
Description:
I have linked the the data input form to the add record button 
I have added the sidebar icon to the navigation bar .. .this will bring the sidebar panel that will contain different options, these are still to be implemented for now i just linked the data input form to it as well to just show the behaviour of the sidebar panel
The field values are captured and saved ...the api to save the data is still to  be implemented 

https://github.com/kartoza/miniSASS/assets/70011086/034d712c-6276-4e68-8ac0-17bf880eb99c

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 4 min 45 sec

File: django_project/minisass_frontend/src/components/DataInputForm/index.tsx [hunk 1]
```diff
@@ -57,15 +62,104 @@ type DataInputFormProps = Omit<
...
+  { label: "m S/m", value: "m S/m" },
+  { label: "Unknown", value: "uknown" },
 ];
 
 const DataInputForm: React.FC<DataInputFormProps> = (props) => {
+
+  // TODO still need to get units and river categories and save data to db
+
+  // State to store form values
+  const [formValues, setFormValues] = useState({
```
Comment (dimasciput) [inline django_project/minisass_frontend/src/components/DataInputForm/index.tsx:88]: Please use [formik ](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwia36W_xsKCAxUXUGwGHfzUB7EQFnoECBsQAQ&url=https%3A%2F%2Fformik.org%2F&usg=AOvVaw1P3YeJxlNpFT15wj_TX7jM&opi=89978449)instead.

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 2 min 53 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 4 min 1 sec

File: django_project/minisass_frontend/src/components/DataInputForm/index.tsx (hunk 1 above)
Comment (tinashechiraya) [reply django_project/minisass_frontend/src/components/DataInputForm/index.tsx:88]: @dimasciput  I have added it

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 4 min 24 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 4 min 39 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 4 min 32 sec

**Comment by tinashechiraya** [comment]
this has been merged via pr #226

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 38 sec

**Pull request closed without merging**